easybuild/easyconfigs/r/Rust/Rust-1.54.0-GCCcore-11.2.0.eb

```diff
@@ -34,6 +34,10 @@ configopts = "--enable-extended --sysconfdir=%(installdir)s/etc "
 # since Rust is a build dependency for cryptography that is included with Python
 configopts += "--set=llvm.ninja=false "
 
+# Use ./x.py to bootstrap so that options like -j N are correctly passed through
+build_cmd = "./x.py build"
```

Comment:
@jfgrimm We should explicitely pass `--jobs` here, and add a comment with a pointer to the docs on `x.py` (since this looks bloody weird without more info):

```python
# see https://rustc-dev-guide.rust-lang.org/building/how-to-build-and-run.html#what-is-xpy
build_cmd = "./x.py build --jobs %(parallel)s"
install_cmd = "./x.py install --jobs %(parallel)s"
```

Reply:
added some additional comments, including clarification that the easyblock already adds `-j %(parallel)s`

Reply:
that said, I think that's only for the build step. Added it explicitly to the install cmd